tests/storage/business/test_watcher.py

```diff
@@ -112,6 +173,100 @@ def test_scan(tmp_path: Path):
     assert call.args[1] is None
 
 
+@pytest.mark.unit_test
+def test_scan_recursive_false(tmp_path: Path, db_session: Session):
+    # Base.metadata.create_all(db_session)
+
+    def count_studies():
+        return db_session.execute("SELECT COUNT(*) FROM study").scalar()
```

Comment:
Minor: you could probably use `db_session.query(Study).count()` as you'll hide SQL